test/test_express_opt.jl

```diff
@@ -35,4 +35,4 @@ op = N⊗X
 @test express(op*state) ≈ express(op)*express(state)
 @test express(op*state) ≈ express(F1⊗Z2)
 
-state = (3im*(2*dagger(Z1)+dagger(Y1))) * (3im*(2*X1+X2))
```

Comment:
Why is this removed from the tests?

Reply:
There is currently no predefined dagger for any of the basis states.

Reply:
But this worked previously, right? Isn't there still just a generic dagger wrapper? One should be able to represent `dagger(ket)` without having to be able to construct a bra. There is no need for predefined daggers of basis states.

Reply:
generally things that work should not be broken -- that would require a much more careful release to make sure that other dependent libraries do not break on such a change

Reply:
I agree, just made the change. Everything should be in working order.